tests/core/potentials_test.py

```diff
@@ -0,0 +1,103 @@
+import jax
+import jax.numpy as jnp
+import numpy as np
+import pytest
+
+from ott.core import Sinkhorn, linear_problems
+from ott.geometry import pointcloud
+from ott.tools import sinkhorn_divergence
+from ott.tools.gaussian_mixture import gaussian
+
+
+class TestEntropicPotentials:
+
+  @pytest.mark.fast.with_args(
+      forward=[False, True], eps=[1e-2, 1e-1], only_fast=0
+  )
+  def test_entropic_potentials(
+      self, rng: jnp.ndarray, forward: bool, eps: float
+  ):
+    n1, n2, d = 64, 96, 2
+    key1, key2, key3, key4 = jax.random.split(rng, 4)
+
+    mean1, mean2 = jnp.zeros(d), jnp.ones(d) * 4
+    cov1, cov2 = jnp.eye(d), jnp.array([[2, 0], [0, 0.5]])
+    g1 = gaussian.Gaussian.from_mean_and_cov(mean1, cov1)
+    g2 = gaussian.Gaussian.from_mean_and_cov(mean2, cov2)
+    x = g1.sample(key1, n1)
+    y = g2.sample(key2, n2)
+
+    geom = pointcloud.PointCloud(x, y, epsilon=eps)
+    prob = linear_problems.LinearProblem(geom)
+    out = Sinkhorn()(prob)
+
+    potentials = out.to_dual_potentials()
+
+    expected_dist = out.reg_ot_cost
```

Comment:
I think the expected_dist in this case should not be out.reg_ot_cost, because that includes the entropic regularization (which is probably hurting your comparison below). 

Instead you could evaluate `jnp.sum(out.matrix * geom.cost_matrix)` and see if the approximation improves.

Reply:
Ok, in that case, for readability, may I suggest to use instead a ratio? like `abs(expected-actual)/expected < 5%` or 10%? (whichever works).